audio-worklet/basic/message-port.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      MessagePort with AudioWorklet | Chrome WebAudio Samples
+    </title>
+    <link rel="stylesheet" href="../../resources/base.css">
+    <script src="../lib/audio-worklet-helper.js"></script>
+  </head>
+  <body>
+    <a href="../index.html" class="link-small">« AudioWorklet home</a>
+    <section>
+      <h1>
+        MessagePort with AudioWorklet
+      </h1>
+      <p>
+        Demonstrates the basic bi-directionaly communication between
+        AudioWorkletNode and AudioWorkletProcessor.
+      </p>
+      <p>
+        <strong>NOTE</strong>: Open up the console to see messages between them.
```

Comment:
A brief description on what is expected in the console would be really helpful to indicate if things are working.

Reply:
Done.